### PR TITLE
Edits

### PR DESCRIPTION
**Pull request by mblue9**
Started editing workshop

### CODE REVIEW

**Comment by mblue9** [comment]
This won't work I guess as it won't find tidySCE but I'll merge & we can fix after.

File: .github/workflows/basic_checks.yaml [hunk 1]
```diff
@@ -44,14 +44,14 @@ jobs:
...
         
-      - uses: docker/build-push-action@v1
-        with:
-          username: ${{ secrets.DOCKER_USERNAME }}
-          password: ${{ secrets.DOCKER_PASSWORD }}
-          repository: stemangiola/biocasia2020_tidytranscriptomics
-          tag_with_ref: true
-          tag_with_sha: true
-          tags: biocasia2020
+      # - uses: docker/build-push-action@v1
```
Comment (stemangiola) [inline .github/workflows/basic_checks.yaml:47]: Do you want me to create the secrets? How come we don't have them this round around?

**Comment by mblue9** [comment]
Yes please add the secrets, we will need them I've just turned off the docker build while we're editing because it adds ~30 mins to the tests here

**Comment by stemangiola** [comment]
> Yes please add the secrets, we will need them I've just turned off the docker build while we're editing because it adds ~30 mins to the tests here

Done!

**Pull request merged**